tabs/applications-setup/browser-setup.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+install_chrome() {
+    printf "%b\n" "Installing Google Chrome..."
+    case $PACKAGER in
+        apt-get)
+            wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+            $ESCALATION_TOOL dpkg -i google-chrome-stable_current_amd64.deb
+            ;;
+        zypper)
+            $ESCALATION_TOOL ${PACKAGER} addrepo http://dl.google.com/linux/chrome/rpm/stable/x86_64 Google-Chrome
+            $ESCALATION_TOOL ${PACKAGER} refresh
+            $ESCALATION_TOOL ${PACKAGER} install google-chrome-stable
+            ;;
+        pacman)
+            git clone https://aur.archlinux.org/google-chrome.git
+            cd google-chrome
```

Comment:
```suggestion
            cd "$HOME/google-chrome"
```
Stop using hardcoded paths.